connector_ebisumart/components/ebisumart_adapter.py

```diff
@@ -71,6 +73,35 @@ def _fetch_data(self, endpoint, params=None):
         if response.status_code == 200:
             return response.json()
 
+    def _get_date_filters(self, last_fetch_date):
+        # Add a one-day buffer
+        buffer_time = timedelta(days=1)
+        last_fetch_date = last_fetch_date - buffer_time
+        user_tz = self.env.user.tz or 'UTC'
+        local_tz = timezone(user_tz)
+        # Convert last_fetch_date from UTC to the user's timezone
+        fetch_date = pytz.utc.localize(last_fetch_date).astimezone(local_tz)
+        last_fetch_date_str = fetch_date.strftime('%Y/%m/%d %H:%M:%S')
+        regist_date_filter = {
+            "column": "REGIST_DATE",
+            "operator": "more_than_equals",
+            "to_char_format": "YYYY/MM/DD HH24:MI:SS",
+            "value": last_fetch_date_str
+        }
```

Comment:
```suggestion
```
I don't think `regist_date_filter` is necessary. Why have this?
The following scenarios are causing problems.
- Create order on 1/12 
- Cancel order on 1/14 
- Import order into Odoo on 1/15
In this case, UPDATE_DATE is 1/14, but REGIST_DATE is 1/12.